### ISSUE TITLE
Without any change, the 3dssd model is run locally, and the results in the validation set are far from those provided by the official model.

### ISSUE THREAD

**Issue by axiaoxiaobai** [open-mmlab/mmdetection3d#1456]
2022-05-03 12:32:35,377 - mmdet - INFO - Environment info:
------------------------------------------------------------
sys.platform: linux
Python: 3.7.9 (default, Aug 31 2020, 12:42:55) [GCC 7.3.0]
CUDA available: True
GPU 0: GeForce GTX 1080 Ti
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 10.1, V10.1.105
GCC: gcc (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609
PyTorch: 1.5.1
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.2 Product Build 20200624 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v0.21.1 (Git Hash 7d2fd500bc78936d1d648ca713b901012f470dbc)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 10.1
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_37,code=compute_37
  - CuDNN 7.6.3
  - Magma 2.5.2
  - Build settings: BLAS=MKL, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -fopenmp -DNDEBUG -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DUSE_INTERNAL_THREADPOOL_IMPL -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_STATIC_DISPATCH=OFF, 

TorchVision: 0.6.0a0+35d732a
OpenCV: 4.1.2
MMCV: 1.4.8
MMCV Compiler: GCC 5.4
MMCV CUDA Compiler: 10.1
MMDetection: 2.23.0
MMSegmentation: 0.23.0
MMDetection3D: 1.0.0rc1+333536f
------------------------------------------------------------

This is the result of the locally trained model on the validation set:

----------- AP11 Results ------------

Car AP11@0.70, 0.70, 0.70:
bbox AP11:92.7432, 90.1055, 88.7083
bev  AP11:90.2209, 88.2416, 82.7446
3d   AP11:89.1342, 78.9447, 76.7021
aos  AP11:92.72, 89.98, 88.47
Car AP11@0.70, 0.50, 0.50:
bbox AP11:92.7432, 90.1055, 88.7083
bev  AP11:92.6534, 90.1333, 89.0516
3d   AP11:92.6314, 89.9677, 88.7398
aos  AP11:92.72, 89.98, 88.47

----------- AP40 Results ------------

Car AP40@0.70, 0.70, 0.70:
bbox AP40:97.4445, 93.2251, 90.2798
bev  AP40:94.6342, 89.7675, 85.7387
3d   AP40:92.4251, 81.8057, 76.2659
aos  AP40:97.42, 93.07, 90.04
Car AP40@0.70, 0.50, 0.50:
bbox AP40:97.4445, 93.2251, 90.2798
bev  AP40:97.2520, 93.9178, 92.1814
3d   AP40:96.9991, 93.5268, 90.4401
aos  AP40:97.42, 93.07, 90.04

**Comment by Tai-Wang** [comment]
Please refer to #612 .

**Comment by Tai-Wang** [comment]
Close due to inactivity.
